index.js

```diff
@@ -4,11 +4,19 @@ var jsEOL = require('node-releases/data/release-schedule/release-schedule.json')
 var path = require('path')
 var e2c = require('electron-to-chromium/versions')
 
+var BrowserslistQuery = require('./query')
 var BrowserslistError = require('./error')
 var env = require('./node') // Will load browser.js in webpack
 
 var FLOAT_RANGE = /^\d+(\.\d+)?(-\d+(\.\d+)?)*$/
 
+// Enum values MUST be powers of 2, so combination are safe
+var QueryType = {
```

Comment:
If you will define `var QUERY_OR = 2; var QUERY_AND = 4;` it will reduce project size ;)